tests/usr.bin/cc/t_ubsan_int_sub_overflow.sh

```diff
@@ -193,12 +190,14 @@ atf_init_test_cases()
 {
 	test_target
 	test $SUPPORT = 'n' && {
-		atf_add_test_case target_not_supported
+		atf_sub_test_case target_not_supported
 		return 0
 	}
-	atf_add_test_case int_sub_overflow
-	atf_add_test_case int_sub_overflow_profile
-	atf_add_test_case int_sub_overflow_pie
-	atf_add_test_case int_sub_overflow_pic
-	atf_add_test_case int_sub_overflow32
+	atf_sub_test_case int_sub_overflow
+	atf_sub_test_case int_sub_overflow_profile
+	atf_sub_test_case int_sub_overflow_pie
+	atf_sub_test_case int_sub_overflow_pic
+	atf_sub_test_case int_sub_overflow32
+    # static option IS NOT supported
```

Comment:
Indentation issue.

While there, isn't there ubsan support for -static?

Reply:
I'm removing it altogether for now. I have the impression from the docs I've read on clang UBSan flags that some flags cannot be used along with static. Anyways, will look at the docs once more and have a final saying soon